.dagger/main.go

```diff
@@ -74,33 +76,82 @@ func (dev *DaggerDev) WithModCodegen() *DaggerDev {
 	return &clone
 }
 
+type Check func(context.Context) error
+
+func (dev *DaggerDev) sdkCheck(sdk string) Check {
+	var base sdkBase
+	switch sdk {
+	case "python":
+		base = &PythonSDK{Dagger: dev}
+	case "go":
+		base = &GoSDK{Dagger: dev}
+	case "typescript":
+		base = &TypescriptSDK{Dagger: dev}
+	case "php":
+		base = &PHPSDK{Dagger: dev}
+	case "java":
+		base = &JavaSDK{Dagger: dev}
+	case "rust":
+		base = &RustSDK{Dagger: dev}
+	case "elixir":
+		base = &ElixirSDK{Dagger: dev}
+	}
+	return func(ctx context.Context) error {
+		if err := base.Lint(ctx); err != nil {
+			return err
+		}
+		if err := base.Test(ctx); err != nil {
+			return err
+		}
+		// FIXME: PublishTest
```

Comment:
:eyes: just wanted to check, is this being brought back before/after we merge this? This does catch real issues, so don't want to leave it too long.

Reply:
Sorry yes, that is WIP, will bring back today

Reply:
Done.